app/api/user_favourite_events.py

```diff
@@ -79,11 +81,52 @@ class UserFavouriteEventDetail(ResourceDetail):
     """
     User Favourite Events detail by id
     """
+    def before_get_object(self, view_kwargs):
+        if 'Authorization' in request.headers:
+            _jwt_required(app.config['JWT_DEFAULT_REALM'])
+        else:
+            raise ForbiddenException({'source': ''}, 'Only Authorized Users can view an event')
```

Comment:
Use decorator

Reply:
@iamareebjamal As discussed earlier, decorators won't work here.

Reply:
Why?

Reply:
@iamareebjamal decorators cannot be declared in function, secondly, we can use has_access, which again requires `current_user`, which is returning null.